src/pages/veelgestelde-vragen.tsx

```diff
@@ -5,31 +5,64 @@ import { FCWithLayout, getLayoutWithMetadata } from '~/components/layout';
 import { MaxWidth } from '~/components-styled/max-width';
 import siteText, { TALLLanguages } from '~/locale/index';
 import { parseMarkdownInLocale } from '~/utils/parse-markdown-in-locale';
-import { getSkipLinkId } from '~/utils/skipLinks';
+
 import styles from './over.module.scss';
 import { Collapsable } from '~/components-styled/collapsable';
 
+import { groq } from 'next-sanity';
+import {
+  getClient,
+  usePreviewSubscription,
+  PortableText,
+  localize,
+} from '~/lib/sanity';
+
+import { targetLanguage } from '../locale/index';
+
+interface IVraagEnAntwoord {
+  vraag: string;
+  antwoord: string;
+  id?: string;
+}
+
 interface StaticProps {
   props: VeelgesteldeVragenProps;
 }
 
 interface VeelgesteldeVragenProps {
   text: TALLLanguages;
   lastGenerated: string;
+  faq: any;
```

Comment:
```suggestion
  faq: Record<string, unknown>;
```
Something like this could probably work too?